tests/suites/1_stateful/05_formats/05_05_parquet/05_05_01_parquet_load_unload.sh

```diff
@@ -29,9 +29,6 @@ test_format() {
 
 	echo "truncate table test_load_unload" | $MYSQL_CLIENT_CONNECT
 
-	curl -s -u root: -XPOST "http://localhost:${QUERY_CLICKHOUSE_HTTP_HANDLER_PORT}" \
```

Comment:
why remove this row?

I forgot that load_unload did not cover copy,  it only tested clickhouse handler and streaming load.
they used diff code path, we should add test for copy.